File: HideUI/app/src/main/java/com/example/hideui/MainActivity.java
package com.example.hideui;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView txt;
    Button show,hide;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txt = findViewById(R.id.textView);
        show = findViewById(R.id.show);
        hide = findViewById(R.id.hide);

        show.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                txt.setVisibility(View.VISIBLE);
            }
        });

        hide.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                txt.setVisibility(View.INVISIBLE);
            }
        });

    }
}

File: KotlinFirstapp/app/src/main/java/com/example/kotlinfirstapp/MainActivity.kt
package com.example.kotlinfirstapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var textview = findViewById<TextView>(R.id.txt)
        textview.text = "0"

        var btn1 = findViewById<Button>(R.id.btn1)  //Increase
        var btn2 = findViewById<Button>(R.id.btn2)  //Reset

        var count : Int = 0;
        btn1.setOnClickListener {
            count++
            textview.text = count.toString()
        }
        btn2.setOnClickListener {
            count = 0
            textview.text = count.toString()
        }
    }
}